docs/ref/contrib/admin/index.txt

```diff
@@ -2220,6 +2217,21 @@ The ``InlineModelAdmin`` class adds:
     inline forms. For example, this may be based on the model instance
     (passed as the keyword argument ``obj``).
 
+.. method:: InlineModelAdmin.has_add_permission(self, request, obj=None)
+
+    The ``has_add_permission`` method is given the HttpRequest and the parent ``obj``.
+    Should return ``True`` if adding an instance is permitted, ``False`` otherwise.
+
+.. method:: InlineModelAdmin.has_change_permission(self, request, obj=None)
```

Comment:
As mentioned in the ticket it is not clear that obj refers to the parent instance, and not the inline instance. The documentation just points to the ModelAdmin.has_add_permission, ModelAdmin.has_change_permission, ModelAdmin.has_delete_permission though the actual use case differs.